### PR TITLE
Add: Repeat Hold Mids in context menu

### PR DESCRIPTION
**Pull request by KaleChips22**

### CODE REVIEW

File: MikuMikuWorld/ScoreContext.cpp [hunk 1]
```diff
@@ -987,6 +987,109 @@ namespace MikuMikuWorld
...
+
+			if (j == sortedSelection.size() - 1)
+			{
+				jPos = findHoldStep(hold, score.notes.at(sortedSelection[0]).ID);
+			}
+
+			for (int i = 1; i < itterations; i++)
+			{
+				int lane = std::clamp(currentRep.lane + i * (patternEnd.lane - patternStart.lane),
+				                      0, 12 - currentRep.width);
```
Comment (sevenc-nanashi) [inline MikuMikuWorld/ScoreContext.cpp:1059]: Lane extension is not respected. Pass `ScoreContext&` to repeatMidsInSelection? (You can refer `ScoreEditorTimeline::noteControl` for implementation

File: MikuMikuWorld/ScoreContext.cpp [hunk 2]
```diff
@@ -987,6 +987,109 @@ namespace MikuMikuWorld
...
+		if (selectedNotes.size() < 3)
+			return;
+
+		Score prev = score;
+
+		Note& note = score.notes[*selectedNotes.begin()];
+		if (!(note.getType() == NoteType::HoldMid || note.getType() == NoteType::Hold))
+			return;
+
+		int startIndex;
```
Comment (sevenc-nanashi) [inline MikuMikuWorld/ScoreContext.cpp:1001]: `holdIndex` sounds more better

**Review by sevenc-nanashi — COMMENTED** [review]
Thank you for your PR! Don't care about translation, I'll add it just before I merge.

File: MikuMikuWorld/DefaultLanguage.h [hunk 3]
```diff
@@ -29,6 +29,7 @@ namespace MikuMikuWorld
 		{ "create_auto_save", "Create Auto Save" },
 		{ "help", "Help" },
 		{ "about", "About" },
+		{ "layer", "Layer" },
```
Comment (sevenc-nanashi) [inline MikuMikuWorld/DefaultLanguage.h:32]: What is this translation for?

**Comment by sevenc-nanashi** [comment]
image

One more thing: Looks like the kind of hold mid follows the first one, but could you make it follow the last one in selection?

File: MikuMikuWorld/DefaultLanguage.h (hunk 3 above)
Comment (KaleChips22) [reply MikuMikuWorld/DefaultLanguage.h:32]: It was not capitalized previously, unlike everything else

Comment (sevenc-nanashi) [reply MikuMikuWorld/DefaultLanguage.h:32]: Makes sense, thanks!

**Review by sevenc-nanashi — APPROVED** [review]
Looks good to me! I'll add small changes, then merge.

**Pull request merged**